.gitignore

```diff
@@ -307,6 +307,8 @@ miniapps/solvers/sol.*
 miniapps/parelag/MultilevelHcurlHdivSolver
 miniapps/parelag/*.mesh
 
+miniapps/elasticity/elasticity
```

Comment:
It's supposed to be a showcase for low level FEOD first and also show AD capabilities. I'm not opposed to a person-name mini app here. Maybe others have an opinion on that.

Reply:
I vote Hooke!

Reply:
I just want to throw that the mechanics community may associate Hooke with linear problems (infinitessimal deformation theory). Since the example presents large deformation theory with the Cauchy stress tensor, maybe Cauchy may be suitable, too? Anyway, I am personally also fine with Hooke as a name. :+1: